specs.sh

```diff
@@ -9,9 +9,9 @@ then
 fi
 
 certoraRun \
-  ./contracts/mvp/CollectibleV1.sol \
-  --verify CollectibleV1:./specs/mvp/CollectibleV1.spec \
-  --packages @openzeppelin=node_modules/@openzeppelin \
+  ./contracts/CollectibleV1.sol \
+  --verify CollectibleV1:./specs/CollectibleV1.spec \
+  --packages @openzeppelin=lib/openzeppelin-contracts/contracts \
```

Comment:
I don't have certora set up yet, so would be cool if you could run this and see if stuff still works as expected @gravityblast